shared/main.tf

```diff
@@ -70,10 +66,6 @@ output "northstar_pipeline" {
   value = "${heroku_pipeline.northstar.id}"
 }
 
-output "rogue_pipeline" {
-  value = "${heroku_pipeline.rogue.id}"
-}
-
```

Comment:
Are these Rogue changes I'm seeing just residual cleanup since merging it into Northstar?

Reply:
Yup! The build for the `shared` module failed because the pipeline no longer exists (since we deleted the apps), and so I removed this to get that workspace back to ✅!